src/trimfastq.py

```diff
@@ -0,0 +1,247 @@
+##################################
+#                                #
+# Last modified 2017/11/08       # 
+#                                #
+# Georgi Marinov                 #
+#                                # 
+##################################
+
+import sys
+import os
+
+# try:
+# 	import psyco
+# 	psyco.full()
+# except:
+# 	pass
+
+def run():
+
+    if len(sys.argv) < 2:
```

Comment:
Using argparse would make this more clear.

Reply:
thanks for suggestion will ask the author to change it.